src/Author.php

```diff
@@ -71,9 +80,15 @@ private function get_author_data( \WP_User $user ) {
 	public function endpoint_args() {
 		return [
 			'id' => [
-				'required' => true,
-				'sanitize_callback' => function ( $author_id, $request, $key ) {
-					return absint( $author_id );
+				'default' => false,
```

Comment:
I think if we use other values for default it will make code more smaller for example if we set the id by default to 0 and then we use: 

```
$id = ( false === $params['id'] ) ? false : absint( $params['id'] );

// can be used as 
$id =  absint( $params['id'] );
```

Also I think we should use `validate_callback` in order to make sure values are either numeric or string.